src/FSharpVSPowerTools/Commands/DepthColorizerManager.cs

```diff
@@ -21,28 +21,37 @@ namespace FSharpVSPowerTools
     [TagType(typeof(DepthRegionTag))]
     public class DepthColorizerTaggerProvider : ITaggerProvider
     {
-        [Import]
-        internal ITextDocumentFactoryService textDocumentFactoryService = null;
 
-        [Import]
-        internal VSLanguageService vsLanguageService = null;
+        private readonly IServiceProvider _serviceProvider;
+        private readonly ITextDocumentFactoryService _textDocumentFactoryService;
+        private readonly ProjectFactory _projectFactory;
+        private readonly VSLanguageService _vsLanguageService;
+
+        [ImportingConstructor]
+        public DepthColorizerTaggerProvider(
+            IServiceProvider serviceProvider,
```

Comment:
Does this actually work? A missing `[Import(typeof(SVsServiceProvider))]` attribute means that MEF can't find the right service provider for composition.